.github/workflows/CI.yml

```diff
@@ -132,25 +136,30 @@ jobs:
           ccache --max-size=2G --set-config cache_dir=~/.ccache
           ccache --show-stats --set-config cache_dir=~/.ccache
 
+      - name: Install cmake
+        uses: aminya/install-cmake@master
+        with:
+          cmake: 3.31.6
```

Comment:
this is unmaintained. See https://github.com/Symbitic/install-cmake/pull/4#issuecomment-2770774769 for recommendation by the author of this action

Reply:
Thanks for helping with this! I found out that we actually didn't need this after upgrading the GEOS version tags to their latest minor.